examples/usr/hello_world/xio_client.c

```diff
@@ -209,12 +210,15 @@ int main(int argc, char *argv[])
 
 	/* create thread context for the client */
 	session_data.ctx = xio_context_create(NULL, 0, -1);
-
+	reg.register_mem = 1;
```

Comment:
Instead of this, consider doing:

```
  if (argc > 3) {
          reg.register_mem = strncmp(..., "rdma", ...) ? 1 : 0;
          ...
  } else {
          reg.register_mem = 1;
          ...
  }
```